dbt/models/output/out_eia__monthly_generators/schema.yml

```diff
@@ -39,6 +39,7 @@ sources:
                   max_value: 0.7
                   row_condition: fuel_type_code_pudl='gas' and report_date>=CAST('2015-01-01' AS DATE) and capacity_factor<>0.0
                   weight_column: capacity_mw
+                  severity: warn
```

Comment:
The tests will still get run, but we'll get a yellow warning rather than a red failure.